wrappers/cis-alarms/versions.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = ">= 0.13.1"
+  required_version = ">= 1.0"
```

Comment:
The code in wrappers should not be updated. The hook autogenerates it.

Reply:
hmm, I'm afraid I don't know how these work. Both the [ALB](https://github.com/terraform-aws-modules/terraform-aws-alb/pull/264) and [ACM](https://github.com/terraform-aws-modules/terraform-aws-acm/pull/122) are failing because of the 0.13 requirement it looks like but each time I run `pre-commit run -a` locally, it changes the `1.0` I specified back to `0.13.1`

Reply:
I will take a look at the failing modules tomorrow morning.

Thank you for an update across all (or almost all)!

Reply:
BTW the version in wrappers is hardcoded right in here - https://github.com/antonbabenko/pre-commit-terraform/blob/master/hooks/terraform_wrapper_module_for_each.sh#L41 . It is ok for the wrappers.